<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tarefas</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Descrição:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateIni">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::YearSection</enum>
   </property>
   <property name="displayFormat">
    <string>yyyy/MM/dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="date">
    <date>
     <year>2023</year>
     <month>10</month>
     <day>18</day>
    </date>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateFim">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy/MM/dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="date">
    <date>
     <year>2023</year>
     <month>10</month>
     <day>18</day>
    </date>
   </property>
  </widget>
  <widget class="QLineEdit" name="caixaDesc">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="caixaPrio">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Selecione</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Baixa</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Média</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Alta</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="caixaObs">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data de Início:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Término:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Responsáveis:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Prioridade:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Observações:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="caixaResp">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>230</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Selecione</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fernando Alves</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mariana Carvalho</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Lucas Eduardo</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btnCad">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>400</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cadastrar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
